{"id":"node_modules/core-js/internals/fails.js","dependencies":[{"name":"C:\\Users\\mibe0521\\Desktop\\yandex_practicum\\yandex_practicum\\package.json","includedInParent":true,"mtime":1663699546899},{"name":"C:\\Users\\mibe0521\\Desktop\\yandex_practicum\\yandex_practicum\\node_modules\\core-js\\package.json","includedInParent":true,"mtime":1663699255162}],"generated":{"js":"module.exports = function (exec) {\n  try {\n    return !!exec();\n  } catch (error) {\n    return true;\n  }\n};\n"},"sourceMaps":null,"error":null,"hash":"955e9409ba89f0e0c771904ef20b7815","cacheData":{"env":{}}}